index.html

```diff
@@ -2162,13 +2132,6 @@ <h3>Presentations</h3>
 <a>holder</a> is expected to either be obtained via the securing mechanism, or
 to not pertain to the <a>validation</a> of the <a>verifiable presentation</a>.
           </dd>
-          <dt><var>proof</var></dt>
```

Comment:
proof can be present in ANY JSON-LD object secured by data integrity, not just the "VerifiablePresentation" RDF class.

Reply:
`proof` is an extension point in the VC specification, just like `credentialStatus` and `issuer`. Conflating Data Integrity and `proof` as the same thing is a mistake. Verifiable Presentations have an extension point for embedded proofs, that extension point is `proof`. Verifiable Credentials have an extension point for embedded proofs, that extension point is `proof`.

-1 to removing it from Verifiable Presentation.